packages/web/src/pages/sign-up-page/pages/SelectArtistsPage.tsx

```diff
@@ -47,10 +49,10 @@ export const SelectArtistsPage = () => {
     (values: SelectArtistsValues) => {
       const { artists } = values
       dispatch(addFollowArtists(artists))
-      // TODO: trigger CTA modal on trending page
       navigate(TRENDING_PAGE)
+      setIsWelcomeModalOpen(true)
     },
-    [dispatch, navigate]
+    [dispatch, navigate, setIsWelcomeModalOpen]
```

Comment:
this is fine for now, but maybe we need this to be triggered by "sign-up-success" saga instead. we can figure out when we do e2e flow